nixos/modules/services/networking/mptcp/mptcp_up_raw

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# A script for setting up routing tables for MPTCP
+
+# can check with (ins)[root@client:~]# journalctl -b -u NetworkManager-dispatcher.service
+# Copy this script into /etc/network/if-up.d/
+
+set -ex
+
+STATUS=$2
+
+# TODO remove
+echo $PATH
+env > /tmp/if_up_env
+
+RT_TABLE=/etc/iproute2/rt_tables
+
+if [ "$DEVICE_IFACE" = "lo" ]; then
```

Comment:
Redundant quotes around lo.

Reply:
Thanks for the comments, I can definitely improve the hook (copied from the mptcp wiki) and will do but it should already allow you to get started with mptcp (which is why I hurried with the PR :p ).